content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+
+## Before you begin
+
+Install Istio by following the instructions corresponding to your platform [installation guide](/docs/setup/). Make sure to enable Ingress Gateway with Kubernetes Ingress support, SDS and [cert-manager](https://docs.cert-manager.io/) optional dependency during installation. Here's an example of how to do this for the [`helm template`](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template) installation path:
+
+{{< text bash >}}
+$ helm template $HOME/istio-fetch/istio \
+  --namespace=istio-system \
+  --set gateways.enabled=true \
+  --set gateways.istio-ingressgateway.enabled=true \
+  --set gateways.istio-ingressgateway.sds.enabled=true \
+  --set global.k8sIngress.enabled=true \
+  --set global.k8sIngress.enableHttps=true \
+  --set global.k8sIngress.gatewayName=ingressgateway \
+  --set certmanager.enabled=true \
+  --set certmanager.email=mailbox@donotuseexample.com \
+  > $HOME/istio-fetch/istio.yaml
+{{< /text >}}
+
+## Configuring DNS name and Gateway
+
+Take a note of the external IP address of the `istio-ingressgateway` service:
+
+{{< text bash >}}
+$ kubectl -n istio-system get service istio-ingressgateway
+{{< /text >}}
+
+Configure your DNS zone so that the domain you'd like to use for this tutorial is resolving to the external IP address of `istio-ingressgateway` service (that you've captured in the previous step). You will need a real domain name for this tutorial in order to get a TLS certificate issued. Let's store the configured domain name into an environment variable for further use:
+
+{{< text bash >}}
+$ INGRESS_DOMAIN=mysubdomain.mydomain.edu
+{{< /text >}}
+
+Our Istio installation contains an automatically generated `Gateway` resource configured to serve the routes defined by the Kubernetes `Ingress` resources. By default it does not use SDS (Secrets Discovery Service), so we need to modify it in order to enable the delivery of the TLS certificates to the `istio-ingressgateway` via SDS:
+
+{{< text bash >}}
+$ kubectl -n istio-system edit gateway
+{{< /text >}}
+
+...and modify the `tls` section corresponding to the `https-default` port as follows:
+
+{{< text yaml >}}
+...
+    tls:
+      mode: SIMPLE
+      privateKey: "use sds"
+      serverCertificate: "use sds"
+      credentialName: "ingress-cert-staging"
+...
+{{< /text >}}
+
+Now it's time to setup a demo application.
+
+## Setting up a demo application
+
+We will be using a simple `helloworld` application for this tutorial. The following command will spin up the `Deployment` and `Service` for the demo application and expose the service using an `Ingress` resource that will be handled by `istio-ingressgateway` (notice the use of `INGRESS_DOMAIN` variable we defined earlier):
```

Comment:
put the notice in an emoji please.

Reply:
I assume you mean a "tip" tag? Fixed. Let me know if I misunderstood this.